cmd/kube-controller-manager/app/storageversionmigrator.go

```diff
@@ -46,7 +46,7 @@ func startSVMController(
 	controllerName string,
 ) (controller.Interface, bool, error) {
 	if !utilfeature.DefaultFeatureGate.Enabled(features.StorageVersionMigrator) ||
-		!clientgofeaturegate.FeatureGates().Enabled(clientgofeaturegate.InformerResourceVersion) {
+		!clientgofeaturegate.InformerResourceVersion2.Enabled() {
```

Comment:
example of in-code usage here.  The exact gate is directly referenced in its package.